guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,247 @@
 [id="host-substatus-overview_{context}"]
 = Host Sub-status Overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
+{Project} contains the following sub-statuses:
 
-The *build* sub-status is relevant for managed hosts.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
```

Comment:
I think the values defined are here:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L4
So https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/config_report.rb#L2

Perhaps the labels are a better way:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L65-L83